CollisionObject[] collision_objects
---
bool isDone
---